apel/db/records/normalised_summary.py

```diff
@@ -62,8 +62,15 @@ def __init__(self):
 
         self._ignored_fields = ["UpdateTime"]
 
-        # This list specifies the information that goes in the database.
-        self._db_fields = self._msg_fields
+        # This list specifies the information that goes in the database. It includes
+        # the additional ServiceLevelType field.
+        self._db_fields = ["Site", "Month", "Year", "GlobalUserName", "VO",
+                            "VOGroup", "VORole", "SubmitHost", "Infrastructure",
+                            "ServiceLevelType",
+                            "NodeCount", "Processors", "EarliestEndTime",
+                            "LatestEndTime", "WallDuration", "CpuDuration",
+                            "NormalisedWallDuration", "NormalisedCpuDuration",
+                            "NumberOfJobs"]
```

Comment:
Is there a particular reason why in this circumstance we add ServiceLevelType in the middle of these fields?  Also, if we're just adding SLT and we're not changing any of the values in the fields after init, could we do this to minimise the risk of typing errors in the init field?
```self._db_fields = self._msg_fields.copy().insert("ServiceLevelType", position_n)```
Would just like to hear your commentary on this really :)

Reply:
This list matches the order of the columns in the database, hence needing to be in the middle.

Doing it programmatically probably would be safer, though I'd want to make it clear where the column is occurring.